### DISCUSSION TITLE
How does the sorting for Tailwind work?

### DISCUSSION THREAD

**Discussion by HenrijsS** [shufo/prettier-plugin-blade#181] [Q&A]
I'm just wondering if you have your own sorter, or are you using [prettier-plugin-tailwindcss
](https://www.npmjs.com/package/prettier-plugin-tailwindcss)?

**Comment by shufo** [comment]
@HenrijsS 
I use my own sorter because the official prettier-plugin-tailwindcss could not integrate it.
